### DISCUSSION TITLE
boost wrapped 多个 model

### DISCUSSION THREAD

**Discussion by kunfei1** [hpcaitech/ColossalAI#5595] [Community | Q&A] [answered]
假如有两个模型都比较大，每个模型单独退推理单卡都不够，训练的时候只微调其中一个，想问一下要如何使用boost函数wrappe了，是需要合并成一个大的模型吗？原始boost函数的模型输入是一个nn.Module类型，不支持模型列表传入

**Comment by kunfei1** [comment]
比如蒸馏模型这种情况用tensor并行要怎么设置呢

**Reply by feixiang7701** [reply]
I have the same question. Have you solved it?

**Reply by kunfei1** [reply]
no..

**Reply by Edenzzzz** [reply]
Have you tried calling booster.boost twice or creating another booster?

**Comment by wangqiang9** [comment]
为啥不支持一下list of nn.Module传入…

**Reply by Edenzzzz** [reply, marked as answer]
You can call boost() on multiple models, but returning a list of list (model, optimizer, criterion, dataloader, lr_scheduler) is convoluted logic and not elegant.
